datadog_checks_dev/datadog_checks/dev/tooling/jira.py

```diff
@@ -0,0 +1,107 @@
+# (C) Datadog, Inc. 2018-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import json
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+from .commands.console import echo_failure
+
+
+class JiraClient:
+    API_URL = 'https://datadoghq.atlassian.net/rest/api'
+    CREATE_ENDPOINT = API_URL + '/3/issue'
+
+    def __init__(self, config):
+        jira_email = config['jira']['user']
+        jira_token = config['jira']['token']
+        if not (jira_email and jira_token):
+            echo_failure('Error: You are not authenticated for Jira. Please set your jira ddev config')
```

Comment:
We try to do all console things in the actual commands, can you move this check there?

Reply:
Sure!